docs/STANDALONE_MODE.md

```diff
@@ -0,0 +1,30 @@
+# Standalone mode
+
+Standalone mode enables plugin launching on different machine than Snap daemon (`snapteld`).
+
+To run plugin in standalone mode it is needed to launch plugin with `--stand-alone` flag, e.g.:
+```
+$ ./snap-plugin-collector-psutil --stand-alone
+```
+Plugin in standalone mode creates HTTP server for communication with Snap framework, by default plugin listens on `8182` port.
+
+It is possible to configure different port for communication with Snap framework using `--stand-alone-port` flag, e.g.:
+
+```
+$ ./snap-plugin-collector-psutil --stand-alone --stand-alone-port 8183
+```
+
+To load plugin in standalone mode it is required to provide URL to indicate a machine
+on which plugin is running (IP address/hostname with port number), e.g.:
+
+```
+$  snaptel plugin load http://127.0.0.1:8182
+```
+
+or
+
+```
+$  snaptel plugin load http://localhost:8182
+```
+
+The rest of operations remains exactly the same as is it for regular plugins.
```

Comment:
Using `regular plugins` might be misleading. Maybe the following is better: `The rest of operations remains exactly the same as is it for plugins running in regular mode.`